projects/pgai/justfile

```diff
@@ -25,7 +25,7 @@ build:
 
 # Install the wheel package locally
 install:
-	@uv sync
+	@uv sync --all-extras
```

Comment:
I was wondering if this is strictly correct, but AFAICT we don't have any optional dependencies at the moment anyway, so it's kind of moot.

Reply:
@JamesGuthrie we do need them, otherwise sqlachemy won't be installed, and you'll get an error when running tests.

Reply:
Aah, I see now. I had missed that we now _do_ have an optional dependency.